final_prep/mod_zero_hero.rb

```diff
@@ -56,29 +76,78 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = {
+  "name" => '',
+  "smell" => '',
+  "weight" => 400,
+  "cities_destroyed" => ['Paris', 'Milan', 'Texas'],
+  "lucky_numbers" => [1, 2, 3, 4],
+  "address" => {
+    "number" => 10,
+    "street" => 'King',
+    "state" => 'CO',
+    "zip" => 80301
+  }
+}
+
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
-
+#   - energy_level = 50
+class SuperHero
+  attr_accessor :name, :super_power, :age
+  attr_reader :arch_nemesis, :power_level, :energy_level
+
+  def initialize(name, super_power, age, power_level, energy_level)
+    @name = name
+    @super_power = super_power
+    @age = age
+    @arch_nemesis = arch_nemesis
+    @power_level = power_level
+    @energy_level = energy_level
+  end
 # - Create the following class methods
 #   - say_name, should print the hero's name to the terminal
 #   - maximize_energy, should update the energy_level to 1000
 #   - gain_power, should take an argument of a number and INCREASE the power_level by that number
-
+  def say_name
+    puts "His name is #{name}"
+  end
+
+  def maximize_energy
+    @energy_level = 1000
+  end
+
+  def gain_power(number)
+    @power_level += number
+    puts "He gained power to #{power_level}"
+  end
+end
 # - Create 2 instances of your SuperHero class
-
-
+learny = SuperHero.new("learny", "Instant learning", 50, 10, 20)
+p learny.name
+p learny.super_power
+p learny.age
+p learny.power_level
+p learny.energy_level
+learny.say_name
+learny.maximize_energy
+learny.gain_power(10)
+p learny.name
+p learny.super_power
+p learny.age
+p learny.power_level
+p learny.energy_level
 # Reflection
 # What parts were most difficult about this exerise?
-
+#Remembering all the technical words and syntax was very difficult.
 # What parts felt most comfortable to you?
-
+#The firsdt part of the exercise was the most easiest to me.
 # What skills do you need to continue to practice before starting Mod 1?
-
+#I need to practice, practice and practice! I defenitly need to get more comfortable qith the wording and understanding the meaning of each line of pseudocode.
```

Comment:
Your annotations looked solid, which makes me believe you do have a good technical understanding. That's awesome! So keep focusing on the technical vocabulary you read and use. More practice will help, and when you are completely immersed in Mod 1, you'll have LOTS of practice :)